src/django/api/tests.py

```diff
@@ -181,7 +181,8 @@ def test_cant_replace_twice(self):
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(
             json.loads(response.content),
-            ['FacilityList 1 has already been replaced.'])
+            ['FacilityList {} has already been replaced.'
+                .format(response_json['id'])])
```

Comment:
Not sure why this was breaking, but it kept trying to use "FacilityList 15 has already been", which made it seem like it was uploading the new list after loading the non-test fixture data.